docs/building-your-bot-on-iadvize.md

```diff
@@ -736,6 +737,62 @@ A quick reply is used for offering several choices to a visitor. Each choice nee
 }
 </pre>
 
+## Implement proactive bot
+
+If you don't need your bot to be proactive, you can ignore this endpoint
+
+Proactive-replies is the endpoint used to proactively engage new visitors with your bot. It should return the first replies you want to send at the start of a conversation, before the first visitor message.
```

Comment:
Add two sentences: 
- when is this endpoint called
- what do we mean by `at the start of the conversation` == > as soon as the visitor opens up the chatbox